math/power/powvialogarithm_test.go

```diff
@@ -0,0 +1,44 @@
+// powvialogarithm_test.go
+// description: Test for PowUseLog
+// author(s) [red_byte](https://github.com/i-redbyte)
+// see powvialogarithm.go
+
+package power
+
+import "testing"
+
+var tests = []struct {
+	name     string
+	base     float64
+	power    float64
+	expected float64
+}{
+	{"0^0", 99, 1, 99},
+	{"-3^9", -3, 9, -19683},
+	{"0^2", 0, 2, 0},
+	{"2^0", 2, 0, 1},
+	{"2^3", 2, 3, 8},
+	{"8^3", 8, 3, 512},
+	{"11^11", 11, 11, 285311670611},
+	{"5^5", 5, 5, 3125},
+	{"-7^2", -7, 2, 49},
+	{"-6^3", -6, 3, -216},
+}
+
+func TestPowUseLog(t *testing.T) {
```

Comment:
reflect the names if you change the name of the PowUseLog function

Reply:
Done